src/osip-monitoring-controller/src/main/resources/Bundle.properties.txt

```diff
@@ -0,0 +1,20 @@
+monitoring.menu.file=File
+monitoring.menu.file.settings=Settings
+monitoring.menu.help=?
+monitoring.menu.help.help=Help
+monitoring.menu.help.about=About
+monitoring.tank.overflowAlarm=Liquid overflow alarm
+monitoring.tank.underflowAlarm=Liquid underflow alarm
+monitoring.tank.tempOverheatAlarm=Overheating alarm
+monitoring.tank.tempUndercoolAlarm=Undercooling alarm
+monitoring.tank.supply=Supply
+monitoring.tank.drain=Drain
+monitoring.tank.fillLevel=Fill level
+monitoring.tank.temperature=Temperature
+monitoring.tank.progress.legend=Progress
+monitoring.tank.progress.x=Time in milliseconds
```

Comment:
Der Witz ist: Die Verläufe rechnen die Zeit bereits in Sekunden, nur der Label war hier falsch.